### DISCUSSION TITLE
Graphic Designer

### DISCUSSION THREAD

**Discussion by parisa4360** [WeAreMahsaAmini/Core#76] [Introductions]
I’m Graphic designer and i can help with all kind of visual that u need

**Comment by parisa4360** [comment]
www.zanzendegiazadi.art

**Comment by syavash** [comment]
The website you've shared is great. Is it Ok to reuse some of the content?

A team is actually working on a website mostly targeting non-Iranians with the goal of awareness. We definitely can use your skills and reusing the content in the website you've shared can help.
